src/EntityManager/DoctrineEntityManager.php

```diff
@@ -15,7 +16,7 @@ class DoctrineEntityManager extends EntityManagerDecorator implements EntityMana
      */
     public static function initialize($conn, Configuration $config, EventManager $eventManager = null)
     {
-        return new self(\Doctrine\ORM\EntityManager::create($conn, $config, $eventManager));
+        return new self(EntityManager::create($conn, $config, $eventManager));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Doctrine\ORM\EntityManager' in method 'initialize'.](https://www.codacy.com/app/Managlea/EntityManager/file/2313601653/issues/source?bid=3149133&fileBranchId=3199776#l19)